src/CassetteOverlay.jl

```diff
@@ -34,7 +28,8 @@ end
 
 function overlay_generator(passtype, fargtypes)
     tt = to_tuple_type(fargtypes)
-    match = _which(tt; method_table=method_table(passtype), raise=false)
+    mt = Base.invoke_in_world(typemax(UInt), method_table, passtype)
```

Comment:
Unfortunately this doesn't seem to work -- it still results in a world age error.
I confirmed `invokelatest` does the same.

@Keno did you have something different in your mind?

Reply:
no, I expected this to work

Reply:
I guess we specifically disallow `invokelatest` in generated functions ...

Reply:
So maybe `typemax(UInt)-1`? 😎

Reply:
No, we disallow switching worlds inside of generated functions entirely, which is probably sensible if a little heavy handed.

Reply:
Well, I guess `invoke_in_world` seems to be disallowed also.